### ISSUE TITLE
Update the color of help text within CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10070]
## Description
Within the CMS, the help text (HEX 757576) does not pass WCAG 2.0 AAA color contrast requirements against the white background

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/182940511-0bdf77dd-3700-4ec1-8c28-ba57416d0f74.png)

## Accessibility Standard

WCAG version 2.0 AAA, [Criterion 1.4.6](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced)

## Acceptance Criteria
- [x] Confirm help text is not used on a background color other than white
- [x] Confirm help text is correctly documented in Design System
- [x] Technical review for updating help text color in CMS to Gray Dark (Hex 595959)
- [x] Change management is not required
- [x] Implementation ticket is created [10442](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10442)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by EWashb** [comment]
@cjterdi can you create the implementation ticket for this one so that we can close it?

**Comment by cjterdi** [comment]
This needs further design review:
The help text color only meets AAAd guidelines with a white background
For help text on "View only" content we are using the base-gray color, but there the help text doesn't meet AAA guidelines on the hover state. To balance usability with accessibility we should consider reviewing the hover state.

**Comment by EWashb** [comment]
Thank you for the clarification. This was one that I remembered from refinement yesterday. Looks like I've prematurely checked off an AC that shouldn't have been. Feel free to uncheck any boxes

**Comment by cjterdi** [comment]
Per discussion with Laura: 
Help text color in MCS: Gray Cool Dark (Hex 525960)
Change Help text in read only formats to Gray Base: (Hex 212121)
We will address hover states as a separate issue along with the help text contrast guidelines. 
The 2nd image (above in the Q&A row) is small body text and should be Gray Base (Hex 212121) I can create a separate ticket for this.

**Comment by cjterdi** [comment]
We did not go through an official change management review because the change has minimal impact.

**Comment by laflannery** [comment]
For tracking purposes, implementation ticket is here: #10442

**Comment by laflannery** [comment]
@BlakeOrgan, this is the original design ticket for the help text.  I'm going to summarize everything we discussed here:

Can you take another look at the help text request above and recommend updates to ensure the text passes 7:1 contrast ratio requirements?

- Currently the help text is `#757575`
- Help text appears mostly on a white `#ffffff` background but does sometimes appear on light blue `#d9e8f6`, red `#f9dede` or gray `#f1f1f1` background.
- Help text can also be disabled, so please also provide direction for disabled text (i.e. should it differ from standard help text).
- Help text is currently size 13px but Dave C believes that is too small and would like this to be reviewed.  Note: there are no size requirements within the WCAG criteria so this is strictly a Design/UX decisions

**Comment by BlakeOrgan** [comment]
Here are my suggested edits and a link to the [Figma file](https://www.figma.com/file/ZENA4mBp6DPyCtvj3NlBRn/Help-Text?node-id=16%3A553) for reference.

Goal: Help text should always pass AAA compliance no matter the background color.

Changes:
- [ ] - Help text will ALWAYS be the same color as the Input label. `#212121`
- [ ] - If an input field is disabled, the input label and help text will stay the same color `#212121`
Reason for change: Keeping the input label and help text the same color at all times ensures that the text is always AAA complaint, even if the text shows up on different background colors. The disabled input field provides enough of a visual indicator that the field is uneditable, so there’s no need for disabled input label and help text.
- [ ] - Increase size of help text from 13 to 14.
- [ ] - Ensure help text link text color is `#004795`, not `#0071BB`. This will ensure that links are always AAA compliant against all background colors.
